README.md

```diff
@@ -52,26 +47,14 @@ Not sure about this one?
 
 `curl -X POST -H "Content-Type: application/json" http://localhost:8118/ingress/message -d <payload>`
 
-### Valid Payloads
```

Comment:
This looks like a duplicate; 

We may remove `http://localhost:8118/ingress/message` so just remove this section from readme